contrib/platform/test/com/sun/jna/platform/win32/Kernel32Test.java

```diff
@@ -539,21 +539,6 @@ public void testCreateProcess() {
         assertTrue(processInformation.dwProcessId.longValue() > 0);
     }
 
-    public void testGetEnvironmentVariable() {
```

Comment:
I think so. I do like that these are some combination of DLL + themed sets though, maybe just Kernel32EnvironmentVariableTest and Kernel32NamedPipeTest and such?